devprofiler/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "devprofiler"
-version = "0.2.1"
+version = "0.2.8"
```

Comment:
Change this 0.2.2. Also, can you delete the extra releases resulting from testing